ansible/roles/passenger/tasks/main.yml

```diff
@@ -22,29 +22,26 @@
   become: true
   when: passenger_repo.changed
 
-- name: install nginx for 20.04
-  apt: pkg=nginx state=latest
-  become: true
-  when: ansible_distribution_version == "20.04"
-
-- name: install passenger module for 16.04
-  apt: pkg=passenger state=latest
-  become: true
-  when: ansible_distribution_version == "16.04"
-
-- name: install passenger module for 20.04
-  apt: pkg=libnginx-mod-http-passenger state=latest
-  become: true
-  when: ansible_distribution_version == "20.04"
-
 - name: install passenger dependencies
   apt: pkg={{ item }} state=latest
   become: true
   with_items:
     - apt-transport-https
     - ca-certificates
     - nginx-extras
-  notify: restart nginx
+
+- name: install nginx and passenger for 20.04
```

Comment:
Just noticed. Let's put "Ubuntu 20.04" in this comment for clarity, and same below.